src/screens/NewsScreen/Content.js

```diff
@@ -86,14 +86,15 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: COLORS.blackgrey
+    color: COLORS.gray500
   },
   title: {
     // marginLeft: 12,
     // marginRight: 12,
     fontFamily: fonts.inter[600],
     fontSize: 14,
-    lineHeight: 17
+    lineHeight: 17,
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text in two different styles has been changed. For the first style, it's changed from `COLORS.blackgrey` to `COLORS.gray500`, and for the second style, a new color `COLORS.white2` has been added. Again, ensure these changes are consistent with your design and accessibility guidelines.

```diff
- color: COLORS.blackgrey
+ color: COLORS.gray500

+ color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->